src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/SentencingMappingResource.kt

```diff
@@ -60,7 +60,7 @@ class SentencingMappingResource(private val mappingService: SentencingMappingSer
       ),
       ApiResponse(
         responseCode = "409",
-        description = "Indicates a duplicate sentencing adjustment mapping has been rejected. If Error code = 409 the body will return a DuplicateErrorResponse",
+        description = "Indicates a duplicate sentencing adjustment mapping has been rejected. If Error code = 1409 the body will return a DuplicateErrorResponse",
         content = [
```

Comment:
This arbitrary error code was intended to allow checking for type of 409 response. It isn't used by the clients - so will look at removing it in the next refactor